src/integrationTest/resources/application-test.yml

```diff
@@ -1,7 +1,7 @@
 flaglist: PF0015,RA0042
 spring:
   datasource:
-    url: ${DB_URL:jdbc:tc:postgresql:11.1:///dbcommondata}
+    url: ${DB_URL:jdbc:tc:postgresql:14:///dbcommondata}
```

Comment:
we can do this later, as we have to run the integrationTests against 11.1 for now?

Reply:
reverted